recipes/open-simulation-interface/config.yml

```diff
@@ -0,0 +1,11 @@
+versions:
+  "3.4.0":
+    folder: "all"
+  "3.3.1":
+    folder: "all"
+  "3.3.0":
+    folder: "all"
+  "3.2.0":
+    folder: "all"
+  "3.1.2":
+    folder: "all"
```

Comment:
Do you need all these versions? Otherwise, I would recommend only adding the latest version. If someone else need an older version, it can be added later. We avoid adding extra unused older versions due storage and CI time.

Reply:
Can we please keep `3.3.1` in there?

Reply:
I need the version 3.1.2. Is it ok to add the versions:
3.4.0
3.3.1
3.1.2 
at once or each separate in different pull request?